Makefile

```diff
@@ -52,6 +52,12 @@ admission-controller:
 apiserver:
 	$(GO) -ldflags '$(LDFLAGS)' -o images/tidb-operator/bin/tidb-apiserver cmd/apiserver/main.go
 
+initializer:
+	$(GO) -ldflags '$(LDFLAGS)' -o images/tidb-operator/bin/tidb-initializer cmd/initializer/main.go
```

Comment:
The `tidb-initializer` name is already taken by the initializer job in the `tidb-cluster` chart

Reply:
fix